packages/devtools/src/utils/ganache-server.ts

```diff
@@ -48,18 +53,15 @@ export class GanacheServer {
 
     const oneMillion = ethers.utils.parseEther("1000000");
 
-    const opts = [`--networkId ${process.env.GANACHE_NETWORK_ID}`, `--port ${this.port}`].concat(
-      accounts.map(a => `--account ${a.privateKey},${a.amount || oneMillion}`)
-    );
-    const cmd = `ganache-cli ${opts.join(" ")}`;
-
-    this.server = spawn("npx", ["-c", cmd], {stdio: "pipe"});
```

Comment:
There was a reason for spawning ganache this way, which I don't fully remember -- unfortunately, the commit message is lost due to a squash commit. It was something like: "When I use `ganache` from `ganache-core`, each test case takes ~2s longer". I couldn't understand why I experienced it at the time. Did you experience anything like that here?

Reply:
Hmm I didn't but perhaps before I go much further with this I should.

Reply:
Maybe just compare the test run times on circle once they pass?

Reply:
I'd be surprised if there was a huge difference, since `ganache-cli` uses `ganache-core` under the hood.